docs/apache-airflow/administration-and-deployment/dagfile-processing.rst

```diff
@@ -19,7 +19,7 @@
 DAG File Processing
 -------------------
 
-DAG File Processing refers to the process of turning Python files contained in the DAGs folder into DAG objects that contain tasks to be scheduled.
+DAG File Processing refers to the process of reading the python files that define your dags and storing them such that the scheduler can schedule them.
```

Comment:
I believe you can find and replace only in specific files (e.g. only in `.rst`)

Reply:
I am not trying to change DAG to dag in this specific PR. I know I could filter just to rst.  It might, in docs, refer to the DAG class or model -- but not really an issue with DAGs.  Anyway, DAGs is more offensive than DAG.  But yes, if time, may tackle DAG rename _later_

Reply:
this is also why i did not tackle it as part of this PR -- it requires a little more thought and examination than this one.  and this one is big enough as it is.